test/integration/scheduler/framework_test.go

```diff
@@ -1023,25 +1011,111 @@ func TestUnreservePlugin(t *testing.T) {
 				if err = wait.Poll(10*time.Millisecond, 30*time.Second, podSchedulingError(testCtx.ClientSet, pod.Namespace, pod.Name)); err != nil {
 					t.Errorf("Expected a scheduling error, but didn't get it. error: %v", err)
 				}
-				if unreservePlugin.numUnreserveCalled == 0 || unreservePlugin.numUnreserveCalled != preBindPlugin.numPreBindCalled {
-					t.Errorf("Expected the unreserve plugin to be called %d times, was called %d times.", preBindPlugin.numPreBindCalled, unreservePlugin.numUnreserveCalled)
+				if reservePlugin.numUnreserveCalled == 0 || reservePlugin.numUnreserveCalled != preBindPlugin.numPreBindCalled {
+					t.Errorf("Expected unreserve to be called %d times, was called %d times.", preBindPlugin.numPreBindCalled, reservePlugin.numUnreserveCalled)
 				}
 			} else {
 				if err = testutils.WaitForPodToSchedule(testCtx.ClientSet, pod); err != nil {
 					t.Errorf("Expected the pod to be scheduled. error: %v", err)
 				}
-				if unreservePlugin.numUnreserveCalled > 0 {
-					t.Errorf("Didn't expect the unreserve plugin to be called, was called %d times.", unreservePlugin.numUnreserveCalled)
+				if reservePlugin.numUnreserveCalled > 0 {
+					t.Errorf("Didn't expect unreserve to be called, was called %d times.", reservePlugin.numUnreserveCalled)
 				}
 			}
 
-			unreservePlugin.reset()
+			reservePlugin.reset()
 			preBindPlugin.reset()
 			testutils.CleanupPods(testCtx.ClientSet, t, []*v1.Pod{pod})
 		})
 	}
 }
 
+// TestReservePluginOrdering tests that invocations of the unreserve
+// operation in reserve plugins is in the reverse order of invocations of
+// the reverse operation.
+func TestReservePluginOrdering(t *testing.T) {
```

Comment:
Can we make these checks part of the above test?

Reply:
Done.